Makefile

```diff
@@ -114,11 +114,17 @@ e2e-test: e2e-build
 		-v $(PWD)/e2e/blob-report:/e2e/blob-report \
 		playwright-e2e
```

Comment:
Nit:

1. can we pull this into a variable to DRY it up so we can keep this in sync with the e2e-build target
2. can we include the project and app name (see how we define IMAGE_NAME in this file) so that the image name doesn't conflict with other projects if the developer has multiple projects that use the platform
3. can we remove "playwright" from the image name to keep the image name more general purpose?


Let's put something like this at the top of the End-to-end (E2E) Testing section right under the header, similiar to the "Release Management" section.

```
# Include project name in image name so that image name
# does not conflict with other images during local development
E2E_IMAGE_NAME := $(PROJECT_ROOT)-$(APP_NAME)-e2e
```

then we can change the e2e-build and e2e-test targets to reference the variable

```suggestion
		$(E2E_IMAGE_NAME)
```

Reply:
Sounds good - but do you set an extra dash after `$(APP_NAME}` - results in double dash `platform-test-nextjs--e2e` here ?

Reply:
Could technically pull all playwright references out into variables like $(E2E_TOOL) or something - ie.. `rm -rf ./e2e/playwright-report`

Reply:
oh, that's because APP_NAME is null so it's doing $(PROJECT_ROOT)--e2e the e2e image doesn't depend on the app right? it has the entire e2e folder. so it should just be $(PROJECT_ROOT)-e2e.

We should probably remove the comment or at least change it to explain that the app isn't relevant e.g.
# The e2e test image includes the test suite for all apps and therefore isn't specific to each app

Reply:
So this now:

```
# Include project name in image name so that image name
# does not conflict with other images during local development.
# The e2e test image includes the test suite for all apps and therefore isn't specific to each app.
E2E_IMAGE_NAME := $(PROJECT_ROOT)-e2e
```